<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RunXun</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="peer_info">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>700</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nobody</string>
    </property>
   </widget>
   <widget class="QListView" name="user_info">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>150</width>
      <height>670</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid</string>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
    </property>
   </widget>
   <widget class="QListView" name="chat_record">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>52</y>
      <width>700</width>
      <height>588</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="input_box">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>640</y>
      <width>700</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid</string>
    </property>
    <property name="text">
     <string>Configuration</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid</string>
    </property>
    <property name="text">
     <string>Information</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="local_port">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>8895</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Your name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>test</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>150</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Send port:
(Can be empty)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="send_port_one">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="send_port_two">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>255</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="send_port_three">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>290</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>340</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
